backend/src/apiserver/model/run.go

```diff
@@ -121,15 +395,29 @@ func (r *Run) GetFieldValue(name string) interface{} {
 	case "DisplayName":
 		return r.DisplayName
 	case "CreatedAtInSec":
-		return r.CreatedAtInSec
+		return r.RunDetails.CreatedAtInSec
+	case "FinishedAtInSec":
+		return r.RunDetails.FinishedAtInSec
 	case "Description":
 		return r.Description
 	case "ScheduledAtInSec":
-		return r.ScheduledAtInSec
+		return r.RunDetails.ScheduledAtInSec
 	case "StorageState":
 		return r.StorageState
 	case "Conditions":
-		return r.Conditions
+		return r.RunDetails.Conditions
+	case "Namespace":
+		return r.Namespace
+	case "ExperimentId":
+		return r.ExperimentId
+	case "State":
+		return r.RunDetails.State
+	case "StateHistory":
+		return r.RunDetails.StateHistory
+	case "PipelineRuntimeManifest":
+		return r.RunDetails.PipelineRuntimeManifest
+	case "RecurringRunId":
+		return r.RecurringRunId
```

Comment:
Since switch-case statement is constant, golang optimizes it during compilation, which results in a much better runtime and memory usage. Moreover, I personally see switch-case more readable. Here is a [handy benchmark](https://stackoverflow.com/questions/46789259/map-vs-switch-performance-in-go).